skill-tree /.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: CI Build
+
+on:
+  pull_request:
+    branches:
+      - '**'
```

Comment:
on all branches?

Why didn't the current action run?

Reply:
Not sure need to check, when I tried running the actions from here https://github.com/Real-Dev-Squad/skill-tree-backend/actions/runs/7197799606/workflow, this is running the intial file which I wrote.
@prakashchoudhary07

Reply:
Have changed this to develop and main